philipmeneghini/InstaConnect
# Frontend Button To Connect to the Backend Added

**Testing Instructions**
cd into the new frontend folder and run `npm install` in order to download dependencies. Then run `npm start` to run the application. Hit the button and "Database Connected!" should be printed below.

https://user-images.githubusercontent.com/63440066/209128662-1f65c5e9-4abe-418d-8be7-d982456259a6.mp4

`CORS` was implemented in the backend to enable browser cross origin connections.
